common/src/main/resources/assets/tardis_refined/models/item/pattern_manipulator.json

```diff
@@ -0,0 +1,140 @@
+				"up": {"uv": [7.5, 4.5, 5.5, 4], "texture": "#0"},
+				"down": {"uv": [7.5, 4, 5.5, 4.5], "texture": "#0"}
+			}
+		},
+		{
+			"from": [5, 6.5, 2],
+			"to": [11, 6.5, 7],
+			"rotation": {"angle": 0, "axis": "y", "origin": [8, 7.5, 1]},
+			"faces": {
+				"north": {"uv": [0, 0, 3, 0], "texture": "#0"},
+				"east": {"uv": [0, 0, 2.5, 0], "texture": "#0"},
+				"south": {"uv": [0, 0, 3, 0], "texture": "#0"},
+				"west": {"uv": [0, 0, 2.5, 0], "texture": "#0"},
+				"up": {"uv": [3, 2.5, 0, 0], "texture": "#0"},
+				"down": {"uv": [3, 0, 0, 2.5], "texture": "#0"}
+			}
+		},
+		{
+			"from": [8, 3.5, 2],
+			"to": [8, 9.5, 7],
+			"rotation": {"angle": 0, "axis": "y", "origin": [8, 6.5, 4.5]},
+			"faces": {
+				"north": {"uv": [3, 0, 5.5, 3], "texture": "#0"},
+				"east": {"uv": [3, 0, 5.5, 3], "texture": "#0"},
+				"south": {"uv": [3, 0, 5.5, 3], "texture": "#0"},
+				"west": {"uv": [3, 0, 5.5, 3], "texture": "#0"},
+				"up": {"uv": [3, 0, 5.5, 3], "texture": "#0"},
+				"down": {"uv": [3, 0, 5.5, 3], "texture": "#0"}
+			}
+		},
+		{
+			"from": [6, 5, 7],
+			"to": [10, 8, 11],
+			"rotation": {"angle": 0, "axis": "y", "origin": [8, 7, 1]},
+			"faces": {
+				"north": {"uv": [3, 3, 5, 4.5], "texture": "#0"},
+				"east": {"uv": [5, 4.5, 3, 6], "texture": "#0"},
+				"south": {"uv": [3, 3, 5, 4.5], "texture": "#0"},
+				"west": {"uv": [3, 4.5, 5, 6], "texture": "#0"},
+				"up": {"uv": [2, 4.5, 0, 2.5], "texture": "#0"},
+				"down": {"uv": [0, 2.5, 2, 4.5], "texture": "#0"}
+			}
+		},
+		{
+			"from": [7, 5, 9],
+			"to": [9, 8, 13],
+			"rotation": {"angle": 45, "axis": "x", "origin": [8, 5, 9]},
+			"faces": {
+				"north": {"uv": [2, 5.5, 3, 7], "texture": "#0"},
+				"east": {"uv": [7.5, 0, 5.5, 1.5], "texture": "#0"},
+				"south": {"uv": [2, 5.5, 3, 7], "texture": "#0"},
+				"west": {"uv": [5.5, 0, 7.5, 1.5], "texture": "#0"},
+				"up": {"uv": [3, 5.5, 2, 3.5], "texture": "#0"},
+				"down": {"uv": [3, 3.5, 2, 5.5], "texture": "#0"}
+			}
+		},
+		{
+			"from": [10, 6, 8],
+			"to": [13, 8, 10],
+			"rotation": {"angle": 22.5, "axis": "z", "origin": [10, 6, 9]},
+			"faces": {
+				"north": {"uv": [7, 1.5, 5.5, 2.5], "texture": "#0"},
+				"east": {"uv": [2, 2.5, 3, 3.5], "texture": "#0"},
+				"south": {"uv": [5.5, 1.5, 7, 2.5], "texture": "#0"},
+				"west": {"uv": [2, 2.5, 3, 3.5], "texture": "#0"},
+				"up": {"uv": [1.5, 7, 0, 6], "texture": "#0"},
+				"down": {"uv": [1.5, 6, 0, 7], "texture": "#0"}
+			}
+		},
+		{
+			"from": [3, 6, 8],
+			"to": [6, 8, 10],
+			"rotation": {"angle": -22.5, "axis": "z", "origin": [6, 6, 9]},
+			"faces": {
+				"north": {"uv": [5.5, 1.5, 7, 2.5], "texture": "#0"},
+				"east": {"uv": [3, 2.5, 2, 3.5], "texture": "#0"},
+				"south": {"uv": [7, 1.5, 5.5, 2.5], "texture": "#0"},
+				"west": {"uv": [3, 2.5, 2, 3.5], "texture": "#0"},
+				"up": {"uv": [0, 7, 1.5, 6], "texture": "#0"},
+				"down": {"uv": [0, 6, 1.5, 7], "texture": "#0"}
+			}
+		}
+	],
+	"display": {
```

Comment:
The displays need work, specifically to do with the GUI and dropped item models. I'd also argue that the item need to be positioned better in the hand.
![image](https://user-images.githubusercontent.com/34910888/212627710-341b478d-e3af-49c7-bade-ad296fb21ddd.png)
![image](https://user-images.githubusercontent.com/34910888/212627913-6945d4bd-7c27-4de6-8325-640b73bedd31.png)

Reply:
Fixed